test/page.spec.js

```diff
@@ -366,15 +366,17 @@ module.exports.addTests = function({testRunner, expect, headless, Errors, Device
         expect(message.type()).toEqual('warn');
     });
     it_fails_ffox('should have location when fetch fails', async({page, server}) => {
+      // The point of this test is to make sure that we report console messages from
+      // Log domain: https://vanilla.aslushnikov.com/?Log.entryAdded
       await page.goto(server.EMPTY_PAGE);
       const [message] = await Promise.all([
         waitEvent(page, 'console'),
         page.setContent(`<script>fetch('http://wat');</script>`),
       ]);
-      expect(message.text()).toContain(`ERR_NAME_NOT_RESOLVED`);
+      expect(message.text()).toContain(`CORS`);
```

Comment:
Does this still go through the log domain?

Reply:
yeah that's still the case